internal/manager/server.go

```diff
@@ -27,6 +27,8 @@ const (
 	allowedChangeRefType = "BRANCH"
 	// letterBytes contains letters to use for random strings.
 	letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	// defaultQueuePollInterval defines the queue pill interval in seconds.
```

Comment:
```suggestion
	// defaultQueuePollInterval defines the queue poll interval in seconds.
```
I assume that there is a typo here

Reply:
Fixed in eeab191.